internal/runnerinstall/k8s.go

```diff
@@ -215,6 +219,13 @@ func (i *K8sRunnerInstaller) InstallFlags(set *flag.Set) {
 			"The required version number format is: 'vX.Y.Z'.",
 	})
 
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-helm-ref",
+		Target: &i.Config.HelmRef,
+		Usage: "The git ref of the Helm chart to use for the Waypoint runner install. " +
+			"The ref may also be a branch name like 'main'. Takes precedent over -k8s-helm-version.",
+	})
```

Comment:
If this is mostly for our internal use, i'd vote for making it a hidden flag so we don't noisify the docs.

```suggestion
Hidden: true,
	})
```